src/test/java/org/javarosa/core/model/TriggerableDagTest.java

```diff
@@ -500,8 +501,8 @@ public void equivalent_predicates_with_function_calls_should_produce_the_same_re
 
     @Test
     public void parsing_forms_with_cycles_by_self_reference_in_calculate_should_fail() throws IOException {
-        exceptionRule.expect(RuntimeException.class);
-        exceptionRule.expectMessage("Dependency cycles amongst the xpath expressions in relevant/calculate");
+        exceptionRule.expect(XFormParseException.class);
```

Comment:
I think it's OK because `XFormParseException` extends `RuntimeException`. It will break  downstream tools only if they expect the original message, which wouldn't be a very good practice in the first place. I guess we can change the message to the old one too if you want.

Reply:
No change needed -- just wanted to have a note that we'd considered it for future reference.